bitcoin/rpc.py

```diff
@@ -51,12 +41,8 @@
 
 DEFAULT_HTTP_TIMEOUT = 30
 
-# (un)hexlify to/from unicode, needed for Python3
-unhexlify = binascii.unhexlify
-hexlify = binascii.hexlify
-if sys.version > '3':
-    unhexlify = lambda h: binascii.unhexlify(h.encode('utf8'))
-    hexlify = lambda b: binascii.hexlify(b).decode('utf8')
+unhexlify = lambda h: binascii.unhexlify(h.encode('utf8'))
+hexlify = lambda b: binascii.hexlify(b).decode('utf8')
```

Comment:
I think using `hexlify`/`unhexlify` with altered semantics (added utf8 encode/decode) might be a base for (small) confusion in the future, particularly because hexlify/unhexlify are used without this added semantics in other places.

I would add encode/decode in place of use for these functions, or named the altered-semantics ones differently.

Granted, that would increase the size of the diff with no difference in actual functionality, so it can be argued that it is better to minimize changes than resolve possible very small confusion. I personally would lean towards removing even small confusion, though.